modules/adliveBidAdapter.js

```diff
@@ -0,0 +1,71 @@
+import * as utils from 'src/utils';
+import { registerBidder } from 'src/adapters/bidderFactory';
+import { BANNER } from 'src/mediaTypes';
+
+const BIDDER_CODE = 'adlive';
+const ENDPOINT_URL = 'https://api.publishers.adlive.io/get';
+const CURRENCY = 'USD';
+const TIME_TO_LIVE = 360;
+
+export const spec = {
+  code: BIDDER_CODE,
+  supportedMediaTypes: [BANNER],
+
+  isBidRequestValid: function(bid) {
+    return !!(bid.params.hashes && utils.isArray(bid.params.hashes));
+  },
+
+  buildRequests: function(validBidRequests) {
+    let requests = [];
+    utils._each(validBidRequests, function(bid) {
+      requests.push({
+        method: 'POST',
+        url: ENDPOINT_URL,
+        options: {
+          contentType: 'application/json',
+          withCredentials: true
+        },
+        data: JSON.stringify({
+          transaction_id: bid.bidId,
+          hashes: utils.getBidIdParameter('hashes', bid.params)
+        }),
+        bid
+      });
+    });
+
+    return requests;
+  },
+
+  interpretResponse: function(serverResponse, bidRequest) {
+    try {
+      const response = serverResponse.body;
+      const bidResponses = [];
+
+      utils._each(response, function(bidResponse) {
+        if (!bidResponse.is_passback) {
+          bidResponses.push({
+            requestId: bidRequest.bidId,
+            cpm: bidResponse.price,
+            width: bidResponse.size[0],
+            height: bidResponse.size[1],
+            creativeId: bidResponse.hash,
+            currency: CURRENCY,
+            netRevenue: true,
+            ttl: TIME_TO_LIVE,
+            ad: bidResponse.content
+          });
+        }
+      });
+
+      return bidResponses;
+    } catch (err) {
+      utils.logError(err);
+      return [];
+    }
+  },
+
+  getUserSyncs: function (syncOptions, serverResponses, gdprConsent) {
```

Comment:
If you don't have any userSync pixels you want to fire, you can just remove the entire `getUserSyncs` property from your adapter.  It's not required to define it.

Reply:
If I do not define this function, tests do not pass.

Reply:
Can you clarify what tests do not pass?  I tried to disabled that part of your adapter code in the branch copy I checked out and ran`gulp test`.  There were no unit tests errors from what I saw.

Reply:
@mifanich I ran the browserstack tests (which is what circleci runs for us automatically) locally with the `getUserSyncs` code disabled and I didn't see any test failures.

It's possible there was some other issue that happened in the circleci tests you saw that failed that were unrelated to your adapter's code.  Can you try to make the change again and push it in this PR so we can retest it?